wagtail/snippets/templates/wagtailsnippets/snippets/edit.html

```diff
@@ -63,6 +63,37 @@
 
             LockUnlockAction('{{ csrf_token|escapejs }}', '{{ action_url }}');
             ActivateWorkflowActionsForEditView('[data-edit-form]');
+
+            {% if publishing_will_cancel_workflow %}
```

Comment:
Again, could we share this rather than copy it, if the logic is identical aside from the view?